### PR TITLE
Analysis to Classify Cancertype

### PR DESCRIPTION
**Pull request by gwaygenomics**
This analysis uses compression algorithm features to classify cancer-type in the PanCanAtlas. There are also a couple of other minor modifications in other scripts in this module. (adjusting function arguments, correcting covariate data, etc.)

### CODE REVIEW

File: 8.tcga-classify/classify-cancer-types.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
+"""
+2018 Gregory Way
+7.tcga-classify/classify-cancer-types.py
+
+Predict cancer-types using elastic net logistic regression and compressed gene
+expression features
+
```
Comment (ajlee21) [inline 8.tcga-classify/classify-cancer-types.py]: what different cancer types?  How many classes do you have? (maybe unnecessary to add, just curious)

File: 8.tcga-classify/classify-cancer-types.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                    # Train the model
+                    print(
+                        "Training model... cancer-type: {}, "
+                        "algorithm: {}, signal: {}, z_dim: {}, "
+                        "seed: {}".format(acronym, alg, signal, z_dim, seed)
+                    )
+
+                    # Fit the model
+                    cv_pipeline, y_pred_train_df, y_pred_test_df, y_cv_df = train_model(
```
Comment (ajlee21) [inline 8.tcga-classify/classify-cancer-types.py]: How robust are your hyperparam fits using CV?

File: 8.tcga-classify/classify-cancer-types.py (hunk 1 above)
Comment (ajlee21) [reply 8.tcga-classify/classify-cancer-types.py]: Is there a reason you chose to us elastic net?

File: 8.tcga-classify/classify-cancer-types.py (hunk 2 above)
Comment (gwaygenomics) [reply 8.tcga-classify/classify-cancer-types.py]: They are fairly variable. And their variability varies a lot depending on the classification task. This analysis takes a while to run, so I sweep over only a few combinations. Ideally I would sweep over more.

**Pull request merged**